homeassistant/components/knx/__init__.py

```diff
@@ -46,6 +48,8 @@
 TUNNELING_SCHEMA = vol.Schema(
     {
         vol.Required(CONF_HOST): cv.string,
+        vol.Optional(CONF_KNX_AUTO_RECONNECT, default=True): cv.boolean,
```

Comment:
Please create a constant for this default value e.g. `DEFAULT_AUTO_RECONNECT`.

Reply:
Done